install.sh

```diff
@@ -6,9 +6,12 @@ if [ "$(uname)" == "Darwin" ]; then
    echo "Done."
```

Comment:
and you forgot to copy `addon.js` in Darwin.😂

Reply:
I don't know the macOS application profile directory, so I can only guess